tools/benchmark/build.gradle.kts

```diff
@@ -28,7 +29,8 @@ android {
 }
 
 dependencies {
-
+    implementation(project(":dd-sdk-android-core"))
```

Comment:
anything from the `core` is used here in the end?

Reply:
The `SpanEvent` generated from json schema has a usage of `JsonSerializer` which points to `core`, do you know if there is a workaround other than depending on `core`?